lms/envs/common.py

```diff
@@ -3916,3 +3920,17 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 ##################### SUPPORT URL ############################
 SUPPORT_HOW_TO_UNENROLL_LINK = ''
+
+################### MONITORING ###############################
+# .. toggle_name: CODE_OWNER_MAPPINGS
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: None
+# .. toggle_description: Used to set monitoring custom metrics for owner. Dict with keys of code owner and value as list of dotted path module names owned by code owner.
+# .. toggle_category: monitoring
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2020-05-29
+# .. toggle_expiration_date: None
+# .. toggle_tickets: None
+# .. toggle_status: supported
+# .. toggle_warnings: None
+CODE_OWNER_MAPPINGS = None
```

Comment:
Can we colocate this new setting in the new monitoring-specific Django app?

Reply:
@nasthagiri: This is actually related to an open toggle documentation question I've had.  The line `CODE_OWNER_MAPPINGS = None` isn't needed, and I actually just added this to give something to annotate.  I _think_ annotations are meant to annotate a declaration, but I have no need of declaring this setting, just using it.   Should I just let these annotations float (i.e. not be next to a line of code) at the top of the file that uses the setting?
FYI: @feanil

Reply:
Since this is a no-line, I just made a similar line in the middleware.py.  See if the adjustment is ok.